### ISSUE TITLE
inputProps: onClick not working

### ISSUE THREAD

**Issue by aroarian** [ericgio/react-bootstrap-typeahead#616]
### Version

5.1.4

### Steps to reproduce

Add inputProps as a prop and add onClick

```
  inputProps={{
    onClick: () => console.log('test')
  }}
```
Once built click on input box

https://codesandbox.io/s/8zw7b-8zw7b

### Expected Behavior

Upon clicking the input box the function should run the code

### Actual Behavior

Upon clicking the input box the function code is not run.

**Comment by ericgio** [comment]
@aroarian: Thanks for the bug report, I can reproduce the issue.

**Comment by GuttermanA** [comment]
Any movement on this? Right now it's not great to have to apply the onClick directly to the renderMenuItemChildren.

**Comment by ericgio** [comment]
Fixed in v5.2.0
